resources/views/bootstrap-4/input.blade.php

```diff
@@ -38,6 +38,8 @@
                     'value' => $isWired ? null : ($value ?? ''),
                     'aria-describedby' => $caption ? $id . '-caption' : null,
                 ]) }}/>
+                <x-form::partials.caption :inputId="$id" :caption="$caption"/>
+                <x-form::partials.error-message :message="$errorMessage"/>
```

Comment:
@Balsakup It seems that feedback message and the caption should indeed be positioned at the same level with the input, but it would be better to position it under the label when it is positioned under the input (input file or floating label).

Reply:
I would accept this PR once this is fixed.

Reply:
@Okipa, feedback will never be shown if `is-invalid` or `is-valid` class is not at the same level

Reply:
It will if you just move the 2 lines under the following part: 

```blade
@if($type === 'file' || (! $prepend && ! $append && $displayFloatingLabel))
    <x-form::partials.label :id="$id"
                                           :class="$type === 'file' ? 'custom-file-label' : 'form-label'"
                                           :label="$type === 'file' ? $placeholder : $label"/>
@endif
```

But it makes no sense to let those lines before the label, it has to be positioned under.